src/server/helpers/search.js

```diff
@@ -246,6 +277,14 @@ search.generateIndex = () => {
 					.map(search.indexEntity)
 			);
 		})
+		.then(() =>
+			 Promise.all(Area.forge()
```

Comment:
No need to surround this with `Promise.all`, I think - the map function should return a promise which resolves once all the elements in the collection have been processed.